qrec/stage_run.py

```diff
@@ -461,8 +327,7 @@ def run_experiment(
                 )
             )
 
-        # Check for jumps in the parameters
-        # TODO: Why 3000? use
+        # Check for jumps in the parameters.
         if qlearning.n0[new_beta_idx] > hyperparam.check_jump_threshold:

```

Comment:
`hyperparm.check_jump_threshold` is set in `experiment/1/000-greedy.py` to `3000`. How was this value chosen?